### PR TITLE
add support for description

### PR DESCRIPTION
**Pull request by vdbg**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -151,6 +153,9 @@ def get_all_devices(self) -> list[Device]:
             logging.info("Refreshing all devices cache")
             self._devices_cache = [Device(x) for x in self.api.list_devices_detailed()]
 
+            for device in self._devices_cache:
+                device.description = self._device_descriptions.get(device.id, None)
```
Comment (Nickolaim) [inline main.py:157]: The second parameter for `dict.get()` is `None`, there is no really need to set it explicitly.

File: main.py [hunk 2]
```diff
@@ -208,7 +213,7 @@ def get_device(self, update: Update, context: CallbackContext) -> Device:
             return device
 
         for alias in self.hubitat.device_aliases:
-            pattern = alias[0]
+            pattern = alias[0].lower()
```
Comment (Nickolaim) [inline main.py:216]: There is `case_insensitive` setting. Should it be used here?

File: main.py [hunk 3]
```diff
@@ -385,15 +392,22 @@ def command_unknown(self, update: Update, context: CallbackContext) -> None:
     def list_devices(self, update: Update, context: CallbackContext, devices: dict[str, Device], title: str):
         self.request_access(update, context, AccessLevel.DEVICE)
         devices_text = []
+
+        def get_description(device: Device) -> str:
+            if device.description:
+                return ": " + device.description
```
Comment (Nickolaim) [inline main.py:398]: `f": {device.description}"`

**Review by Nickolaim — APPROVED** [review]

File: main.py (hunk 1 above)
Comment (vdbg) [reply main.py:157]: duh. thx

File: main.py (hunk 3 above)
Comment (vdbg) [reply main.py:398]: What makes it a better alternative?

**Pull request merged**